source/D0323r5.bs

```diff
@@ -387,168 +373,167 @@ this constructor shall be a constexpr constructor.
 constexpr expected(expected && rhs) noexcept(see below);
 ```
 
-*Effects*: If `rhs` contains a value initializes the contained value as if
+*Effects*: If `bool(rhs)`, initializes `val` as if
 direct-non-list-initializing an object of type `T` with the expression
 `std::move(*rhs)` (if `T` is not `void`).
 
-If `rhs` does not contain a value initializes the unexpected value as if
+If `! bool(rhs)`, initializes `val` as if
 direct-non-list-initializing an object of type `unexpected<E> ` with the
-expression `std::move(unexpected(rhs.error()))`.
+expression `unexpected(std::move(rhs.error()))`.
 
 `bool(rhs)` is unchanged.
 
 *Postconditions*: `bool(rhs) == bool(*this)`.
 
-*Throws*: Any exception thrown by the selected constructor of `T` if `T` is not
-`void` or by the selected constructor of `unexpected<E>`.
+*Throws*: Any exception thrown by operations specified in the effect clause.
 
 *Remarks*: The expression inside `noexcept` is equivalent to:
-`is_nothrow_move_constructible_v<T> or `T `is ``void` and
-`is_nothrow_move_constructible_v<E>`.  This constructor shall not participate in
+`is_nothrow_move_constructible_v<T>` or `T `is `void` and
+`is_nothrow_move_constructible_v<E>`. This constructor shall not participate in
 overload resolution unless `is_move_constructible_v<T> and
-is_move_constructible_v<E>`.  If `is_trivially_move_constructible_v<T>` is
+is_move_constructible_v<E>`. If `is_trivially_move_constructible_v<T>` is
```

Comment:
Missing code-quotes here (the `and` is code-quotes as-is).

Reply:
Done